HW_1/Task2.py

```diff
@@ -0,0 +1,15 @@
+"""
+Задача 2: Найдите сумму цифр трехзначного числа.
+
+*Пример:*
+
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) |
+"""
+
+a = int(input("Введите 3-хзначное число: "))
+
+if 99 < a and a < 1000:
+    print(a // 100 + (a // 10) % 10 + a % 10)
```

Comment:
Приоритет у операций деления одинаковый
```suggestion
    print(a // 100 + a // 10 % 10 + a % 10)
```